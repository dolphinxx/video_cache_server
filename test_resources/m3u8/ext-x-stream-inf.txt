#EXTM3U
#EXT-X-STREAM-INF:PROGRAM-ID=1,BANDWIDTH=800000,RESOLUTION=1080x608
1000k/hls/1.m3u8
#EXT-X-STREAM-INF:PROGRAM-ID=2,BANDWIDTH=900000,RESOLUTION=1080x608
/1000k/hls/2.m3u8
#EXT-X-STREAM-INF:PROGRAM-ID=3,BANDWIDTH=1000000,RESOLUTION=1080x608
https://media.anywhere.com/1000k/hls/3.m3u8