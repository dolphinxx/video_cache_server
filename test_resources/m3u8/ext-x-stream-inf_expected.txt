#EXTM3U
#EXT-X-STREAM-INF:PROGRAM-ID=1,BANDWIDTH=800000,RESOLUTION=1080x608
http://127.0.0.1:8888/?url=https%3A%2F%2Ftest.com%2Fm3u8%2F1000k%2Fhls%2F1.m3u8
#EXT-X-STREAM-INF:PROGRAM-ID=2,BANDWIDTH=900000,RESOLUTION=1080x608
http://127.0.0.1:8888/?url=https%3A%2F%2Ftest.com%2F1000k%2Fhls%2F2.m3u8
#EXT-X-STREAM-INF:PROGRAM-ID=3,BANDWIDTH=1000000,RESOLUTION=1080x608
http://127.0.0.1:8888/?url=https%3A%2F%2Fmedia.anywhere.com%2F1000k%2Fhls%2F3.m3u8